### DISCUSSION TITLE
Ressources autorisées - évaluation 1

### DISCUSSION THREAD

**Discussion by JONATHAN-THIEBAUD** [heig-vd-dai-course/heig-vd-dai-course#420] [Q&A] [answered]
Bonjour, 
 
Pour l'évaluation de lundi, nous avons accès à notre dépôt git et à la doc Java en plus de notre feuille recto comme les années précédentes ? 
 
Merci d'avance et bonne soirée. 

Jonathan T.

**Comment by ludelafo** [comment, marked as answer]
Bonjour,

Non, les modalités ont changé cette année. 

La présentation faite cette année est ce qui fait foi pour l'évaluation. Vous pouvez la retrouver ici : https://github.com/heig-vd-dai-course/heig-vd-dai-course/tree/main/08-evaluation-1.

Cela vous permettra de préparer et utiliser la feuille de résumé à laquelle vous aurez aussi droit lors de l'examen à la fin du semestre. 

Si j'ai répondu à votre question, n'hésitez pas à marquer ce post comme tel. Je laisserai cette discussion ouverte si d'autres personnes ont la même question.

Une bonne journée à vous aussi,
Ludovic
